### ISSUE TITLE
Where'd everyone go?

### ISSUE THREAD

**Issue by brotherdust** [intelsdi-x/snap#1759]
Should I be worried that snap hasn't seen a commit in a few months now? Where did everyone go?

**Comment by brotherdust** [comment]
No. I was just worried that there hasn't been much commit activity recently. Wondered if the community lost interest. That's all. =)

**Comment by ofrzeta** [comment]
Or Intel for that matter. Definitely feels like that. Everybody disappeared somewhere around Nov/Dec 2017.

**Comment by brotherdust** [comment]
According to what the dev team told me, it's considered a mature product. Implication is that it doesn't need as much development resources as it did before.
